File: 02Algorithm_Study/02_week_homeWork/stock_price.cpp
#include <string>
#include <vector>
#include <queue>


using namespace std;

vector<int> solution(vector<int> prices) {
    vector<int> answer;
    for(int i = 0; i < prices.size(); i++){
        int result = 0;
        for(int j = i + 1; j < prices.size(); j++){
        	if(prices[j]>=prices[i]){
        		result ++;
        	}
        	else{
        		result++;
        		break;
            }
        }
        answer.push_back(result);
    }
    return answer;
}

File: 02Algorithm_Study/02_week_homeWork/stock_price.py
def solution(prices):
    answer = []
    
    for i in range(len(prices)):
        result = 0
        for j in range(i + 1, len(prices)):
            if(prices[i]<=prices[j]):
                result += 1
            else:
                result += 1
                break
        answer.append(result)
    return answer
